assets/src/scripts/a11y.js

```diff
@@ -26,4 +26,48 @@ document.addEventListener( 'DOMContentLoaded', function () {
 	addAttribute( 'div.updated', 'role', 'status' );
 	addAttribute( 'div.notice', 'role', 'status' );
 	addAttribute( 'div.error', 'role', 'alert' );
+
+	// Add aria-label attribute to color picker slider
+	// https://github.com/Automattic/Iris/issues/69
+	addAttribute( 'div.iris-slider.iris-strip', 'aria-label', 'Gradient selector' );
+	addAttribute( '.ui-slider-handle', 'aria-label', 'Gradient selector' );
```

Comment:
English strings should be made i18n compatible.

Let's try: https://pascalbirchler.com/internationalization-in-wordpress-5-0/